workshops/weather_grapher/readme.md

```diff
@@ -0,0 +1,397 @@
+```HTML
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.4/dist/Chart.min.js"></script>
+```      
+Add the next two `<script>` tags at the end of the body. These are simply some Javascript packages that Bootstrap requires to function. 
+```HTML    
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
+```    
+If you want to read about everything that Bootstrap has to offer, the [documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/) is the best place to start. 
+
+## Building the User Interface   
+Inside of our `<body>` tag, add a `<h1>` tag with property `style="text-align:center;"` that displays "Weather Grapher" with `class="mt-3"`. The class adds a slight margin to the top so it doesn't look too cramped on the screen. 
+
+Add a `<div>` after the header with the styling `style="display: flex; justify-content: center;"`, which centers it horizontally on the screen. This `<div>` will hold the input fields for the city, start date, and end date. Inside this `<div>` add an `<input>` with the following properties: `type="text" class="form-control mr-1" id="city" placeholder="City Name" style="width; 200px"`. Repeat this type of `<input>` two more times, except replacing the id and placeholder attributes with "Start Date" and "End Date" respectively.    
+
+```HTML
+<body>
+  <h1 style="text-align: center;" class="mt-3"> Weather Grapher </h1>
+  <div style="display: flex; justify-content: center;">
+    <input type="text" class="form-control mr-1" id="city" placeholder="City Name" style="width: 200px;">
+    <input type="text" class="form-control mr-1" id="startDate" placeholder="Start Date (YYYY-MM-DD)" style="width: 210px;">
+    <input type="text" class="form-control mr-1" id="endDate" placeholder="End Date (YYYY-MM-DD)" style="width: 210px;">
+  </div> 
+</body>
+```   
+The `form-control` class applies some preset styling to make the input field look sleek and the mr-1 class adds a slight right margin. The `placeholder` attribute adds preset text to the input field when nothing is entered in it. We also added (YYYY-MM-DD) to the placeholder because that it is the only date format that the API will accept, which we'll see later when we look at the documentation. We also made the date input fields a little longer so the placeholder text is able to fit.  
```

Comment:
```suggestion
The `form-control` class applies some preset styling to make the input field look sleek and the `mr-1` class adds a slight margin to the right. The `placeholder` attribute adds preset text to the input field when nothing is entered in it. We also added (YYYY-MM-DD) to the placeholder because that it is the only date format that the API will accept, which we'll see later when we look at the documentation. We also made the date input fields a little longer so the placeholder text is able to fit.  
```